docs/tutorials/ssp_profile_catalog_authoring/ssp_profile_catalog_authoring.md

```diff
@@ -127,32 +163,54 @@ Access control policy and procedures address the controls in the AC family that
 <!-- The content here will then replace what is in the profile for this control, after running profile-assemble. -->
 <!-- The added parts in the profile for this control are below.  You may edit them and/or add new ones. -->
 <!-- Each addition must have a heading of the form ## Control my_addition_name -->
 <!-- See https://ibm.github.io/compliance-trestle/tutorials/ssp_profile_catalog_authoring/ssp_profile_catalog_authoring for guidance. -->
 
-## Control ImplGuidance
+## Control Implementation Guidance
 
 Do it carefully.
 
-## Control my_guidance
+## Control Expected Evidence
 
-This is my_guidance.
+Detailed logs.
 
-## Control ExpectedEvidence
+## Control Needed Extra
+
+Add prose here for required Section: Needed Extra
+
+## Control A Guidance
+
+This is A Guidance.
+
+## Control B Guidance
+
+This is B Guidance.
 
-Detailed logs.
 ```
 
-In the above markdown example, the fixed, uneditable parts of the control are output first, follwed by a separate section marked, `Editable Content`.  And below the editable content are the individual `Adds` that the profile makes, with each one marked by a header of the form, `## Control guidance_name`.  You may edit the editable content and you may add new Control guidance headers with your own new content. Please refer to Markdown Specifications for Editable Content section to learn more on which headers are valid in Trestle. Then the command, `trestle author profile-assemble --name original_profile --markdown markdown_dir --output new_profile` will create a new OSCAL profile based on the original profile (specified) and the editable content in each control.
+In the above markdown example, the fixed, uneditable parts of the control are output first, follwed by a separate section marked, `Editable Content`.  And below the editable content are the individual `Adds` that the profile makes, with each one marked by a header of the form, `## Control guidance_name`.  You may edit the editable content and you may add new Control guidance headers with your own new content. Please refer to Markdown Specifications for Editable Content section below to learn more on which headers are valid in Trestle. Then the command, `trestle author profile-assemble --name original_profile --markdown markdown_dir --output new_profile` will create a new OSCAL profile based on the original profile (specified) and the editable content in each control.
 
 In a cyclic operation of `generate-edit-assemble` you would simply be re-writing from and to the same json profile, in which case the `--name` is optional and you can just use `trestle author profile-assemble --markdown profile_md --output my_profile`.  This will assemble the markdown profile contents in directory `profile_md` into a json profile named `my_profile` but it will first use the existing `my_profile` json file as the parent profile and incorporate changes (due to user edits) in the markdown version of the profile.
```

Comment:
is --name optional if we want to insert content into exiting profile json? I believe as per description above for catalog if --name is specified then assemble will insert content into that json. -o only specifies the output directory. It can be same as existing one with --name in which case the json would be overwritten, or a new one in which case a new json would be generated. 

In catalog and SSP existing json is specified with --name, but here for profile it is written it will use the one in -o directory. The semantics are getting changed for profile.

I believe the same interpretation is there for SSP assemble also below.

Reply:
In all cases during assembly the --name represents an existing json into which new content is inserted in memory - and the final content is written to --output.  But the user may set --output to be that same original json file - or a different one.  This covers all options in a consistent manner.  I will clarify this in the final section where the options are described - along with the diagram.

Reply:
I have rewritten the section talking about cyclic operation.  profile-assemble requires a parent json file, so if --name is not specified, the value for --output is used for --name.  This was not clear before but should be now.